api/src/controllers/elasticsearch/referent.js

```diff
@@ -63,6 +65,48 @@ async function buildReferentContext(user) {
     });
   }
 
+  if (user.role === ROLES.ADMINISTRATEUR_CLE) {
+    /*
+      Can see:
+      - all referents dep of the department of his etablissement
+      - all ref ADMIN CLE of his etablissement
+      - all ref Classe of his etablissement
+    */
+    const refIds = [];
+    const etablissement = await EtablissementModel.findOne({ chefIds: user._id });
+    if (!etablissement) return { referentContextError: { status: 404, body: { ok: false, code: ERRORS.NOT_FOUND } } };
+    const classes = await ClasseModel.find({ etablissementId: etablissement._id });
+    refIds.push(...classes.flatMap((c) => c.classeReferentIds), ...etablissement.chefIds, ...etablissement.sousChefIds);
```

Comment:
J'ai inversé pour plus de cohérence avec le modèle `Etablissement`

```suggestion
    refIds.push(...classes.flatMap((c) => c.referentClasseIds), ...etablissement.referentEtablissementIds, ...etablissement.coordinateurIds);
```